docs/source/libraries/database/index.rst

```diff
@@ -12,10 +12,14 @@ Description
 
 `Database` is a library for handling different database operations.
 
-The library extends `robotframework-databaselibrary`_.
+This can allow you to query your database after an action has been made to verify the results.
```

Comment:
Is this the main use case for us? Shouldn't it be about making those actions instead?